README.md

```diff
@@ -1 +1,49 @@
-# portfolio-mobile-first
\ No newline at end of file
+
```

Comment:
- I understand you've put in a lot of effort into this, but remember this guidelines are put in place to help you gain the habit of working professionally.

- [ ] On line one, please add the headers will the screenshot attached: Every one has to be included that way. try and copy(edit) the [README](https://github.com/microverseinc/readme-template) provided 👍 
![image](https://user-images.githubusercontent.com/50990925/212017921-99362531-e955-4cb4-9926-edb5831c480b.png)